src/gui/transferlistwidget.cpp

```diff
@@ -1449,3 +1443,10 @@ void TransferListWidget::wheelEvent(QWheelEvent *event)
 
     QTreeView::wheelEvent(event);  // event delegated to base class
 }
+
+void TransferListWidget::openPreviewSelectDialog(const BitTorrent::Torrent *torrent) {
+    auto *dialog = new PreviewSelectDialog(this, torrent);
+    dialog->setAttribute(Qt::WA_DeleteOnClose);
+    connect(dialog, &PreviewSelectDialog::readyToPreviewFile, this, &TransferListWidget::previewFile, Qt::QueuedConnection);
```

Comment:
It makes sense to add a comment on what `Qt::QueuedConnection` is used for. Otherwise, someone might delete it by accident.